euth/user_management/forms.py

```diff
@@ -47,7 +46,7 @@ def clean_password_repeat(self):
 
     def clean_username(self):
         username = self.cleaned_data.get('username')
-        user_exists = User.objects.filter(
+        user_exists = get_user_model().objects.filter(
```

Comment:
Instead of calling `get_user_model()` several times you can also place this line at the top of the module:

```
User = get_user_module()
```

This way all code using the `User` model stays the same. 😉